keyboards/ploopyco/trackball_thumb/trackball_thumb.c

```diff
@@ -0,0 +1,251 @@
+/* Copyright 2021 Colin Lam (Ploopy Corporation)
+ * Copyright 2020 Christopher Courtney, aka Drashna Jael're  (@drashna) <drashna@live.com>
+ * Copyright 2019 Sunjun Kim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "trackball_thumb.h"
+
+#ifndef OPT_DEBOUNCE
+#    define OPT_DEBOUNCE 5  // (ms) 			Time between scroll events
+#endif
+#ifndef SCROLL_BUTT_DEBOUNCE
+#    define SCROLL_BUTT_DEBOUNCE 100  // (ms) 			Time between scroll events
+#endif
+#ifndef OPT_THRES
+#    define OPT_THRES 150  // (0-1024) 	Threshold for actication
+#endif
+#ifndef OPT_SCALE
+#    define OPT_SCALE 1  // Multiplier for wheel
+#endif
+#ifndef PLOOPY_DPI_OPTIONS
+#    define PLOOPY_DPI_OPTIONS { 1200, 1600, 2400 }
```

Comment:
I would really suggest having the default DPI options include lower sensitivity settings - I found it far too sensitive out of the box even on the lowest sensitivity and others on reddit have reported the same. I currently set mine to this:

```
#define PLOOPY_DPI_OPTIONS { 400, 600, 900, 1200, 1600 }
```

Five is possibly too many to cycle through, so perhaps:
```
#define PLOOPY_DPI_OPTIONS { 600, 900, 1200, 1600 }
```

Reply:
```suggestion
#define PLOOPY_DPI_OPTIONS { 600, 900, 1200, 1600 }
```